bndtools.core/bndtools.shared.bndrun

```diff
@@ -13,46 +13,9 @@ crossplatform.runprops: \
 	osgi.console=,\
 	osgi.instance.area.default=../bndtools.test/workspace,\
 	osgi.console.enable.builtin=false
-	
+
+# Keep sorted so that we can diff
 -runrequires: \
-	bnd.identity;id='com.google.guava',\
-	bnd.identity;id='org.apache.ant',\
-	bnd.identity;id='org.apache.felix.gogo.shell',\
-	bnd.identity;id='org.apache.felix.gogo.runtime',\
-	bnd.identity;id='org.apache.felix.gogo.command',\
-	bnd.identity;id='org.eclipse.ui.console',\
-	bnd.identity;id='org.eclipse.equinox.console',\
-	bnd.identity;id='org.eclipse.emf.ant',\
-	bnd.identity;id='org.eclipse.emf.workspace.ui',\
-	bnd.identity;id='org.eclipse.emf.workspace',\
-	bnd.identity;id='org.eclipse.emf.importer.java',\
-	bnd.identity;id='org.eclipse.emf.importer.ecore',\
-	bnd.identity;id='org.eclipse.emf.importer',\
-	bnd.identity;id='org.eclipse.emf.exporter',\
-	bnd.identity;id='org.eclipse.emf.edit.ui',\
-	bnd.identity;id='org.eclipse.emf.databinding',\
-	bnd.identity;id='org.eclipse.emf.databinding.edit',\
-	bnd.identity;id='org.eclipse.emf.codegen.ecore.ui',\
-	bnd.identity;id='org.eclipse.emf.codegen.ui',\
-	bnd.identity;id='org.eclipse.emf.common.ui',\
-	bnd.identity;id='org.eclipse.equinox.event',\
-	bnd.identity;id='org.eclipse.ant.core',\
-	bnd.identity;id='org.eclipse.ant.ui',\
-	bnd.identity;id='org.eclipse.ui.ide.application',\
-	bnd.identity;id='org.eclipse.ui.ide',\
-	bnd.identity;id='org.eclipse.sdk',\
-	bnd.identity;id='org.eclipse.pde.ua.ui',\
```

Comment:
why did you pull pde? I think it should remain for the bndtools.pde project, no?

Reply:
re-added!